tests/e2e/management/test_management.py

```diff
@@ -463,7 +464,7 @@ def test_trash_model_version(self, environment):
         # THEN expect none of its versions to be fetched anymore
         self.wait_for_containers([], model.fetch_model_versions_table)
 
-    @backoff.on_exception(backoff.expo, Exception, max_time=5)
+    @backoff.on_exception(partial(backoff.expo, base=4), Exception, max_time=5)
```

Comment:
This will yield longer wait times for the container to refresh

Reply:
Ouch, we've to start to think about introducing proper "eventually" checking -> Already in JIra